app.js

```diff
@@ -33,7 +33,7 @@ app.use((err, req, res, next) => {
     else next(err)
   })
   
-  app.use((err, rew, res, next) => {
+  app.use((err, req, res, next) => {

```

Comment:
there are a lot of PSQL errors

you don't need a new error handler for each one!

you can check the `err.code` with if/else/switch blocks in one controller

Reply:
Left this in as I can't find the specific code errors for two of my tests, which fail when I take them out.